uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/RESTEditorAdvancedTest.java

```diff
@@ -174,113 +157,190 @@ public void testRestConfiguration() {
 
 		// delete & verify
 		restEditor.deleteRestConfiguration();
-		restEditor.save();
-		workaround_3060();
+		if (restEditor.isDirty())
+			restEditor.save();
 		errorCollector.checkThat(getXPathValue("restConfiguration/@*").isEmpty(), equalTo(true));
 		collectFuseErrors();
 
 		// add & verify
 		restEditor.addRestConfiguration();
-		restEditor.save();
-		workaround_3060();
+		if (restEditor.isDirty())
```

Comment:
even for single line statement i think it is better to use { } to have a more readable code